tests/plugins/gaw/test_dms.py

```diff
@@ -58,12 +83,12 @@ def test_vmrdms_ams(data_vmrdms_ams_cvo):
     vmrdms = stat["vmrdms"]
     assert_allclose(
         [vmrdms.mean(), vmrdms.std(), vmrdms.max(), vmrdms.min()],
-        [185.6800736155262, 237.1293922258991, 2807.6, 5.1],
+        [1.856801e14, 2.371294e14, 2.807600e15, 5.100000e12],
```

Comment:
This are all changes by an order of magnitude of 10^12. Just want to make sure you're aware of this and there is a reason for it.

Reply:
Units, I guessed.